examples/bastion-tunnel-monorepo/.ize/env/testnut/main.tf

```diff
@@ -0,0 +1,133 @@
+resource "aws_key_pair" "root" {
+  key_name   = var.ec2_key_pair_name
+  public_key = var.ssh_public_key
+
+  lifecycle {
+    ignore_changes = [
+      public_key
+    ]
+  }
+}
+
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 2.0"
+
+  name = "${var.env}-vpc"
+  cidr = "10.30.0.0/16"
+
+  azs = [
+    "us-east-1a"
+  ]
+  public_subnets = [
+    "10.30.10.0/23"
+  ]
+
+  private_subnets = [
+    "10.30.20.0/23"
+  ]
```

Comment:
Can we have just public subnet for the sake of simplicity?

Reply:
It's not necessary to show the working concept. It would make sense from the security standpoint though.
If we move it to public, let's make sure to lock down all incoming ports (even 22).